### PR TITLE
Code refractor and making standalone frontend

### PR DESCRIPTION
**Pull request by sanbornsen**

### CODE REVIEW

**Comment by sayanchowdhury** [comment]
Looks good to me. This branch has conflicts so can you rebase on top of the latest commits?

**Comment by sanbornsen** [comment]
Okay. I'll do that.

File: waartaa/client/app/containers/chatapp/chatmenu.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Drawer from 'material-ui/Drawer';
+import {List, ListItem} from 'material-ui/List';
+import Divider from 'material-ui/Divider';
+
+
+var ChatMenu = React.createClass({
+	render: function() {
+		return <Drawer open={true} zDepth={1}>
```
Comment (rtnpro) [inline waartaa/client/app/containers/chatapp/chatmenu.jsx]: Can't this entire block be shifted to 4 spaces towards the left? Are you following some JS styleguide for this? If yes, point me to that.

**Comment by rtnpro** [comment]
@sayanchowdhury shouldn't we follow the same nomenclature as other projects, as in, use dir/file names for the form 'CamelCase'. Also, why are we using, what seems like to be, 8 spaces for a `TAB`. Let's follow the standard 4 spaces for a `TAB`.

**Comment by sanbornsen** [comment]
Yeah, I use tabs for indentation. I'll replace them with 4 spaces. I'll take care of the file names to follow.

**Comment by sayanchowdhury** [comment]
@rtnpro 2 spaces is what I follow. Airbnb has a good JS [style guide](https://github.com/airbnb/javascript)

Let's follow this guide

File: waartaa/client/app/containers/chatapp/chatmenu.jsx (hunk 1 above)
Comment (sanbornsen) [inline waartaa/client/app/containers/chatapp/chatmenu.jsx]: Yeah, my sublime setup shows 4 space for a tab. So, I follow that way. I didn't realize it became 8 space here.

**Comment by sanbornsen** [comment]
Good to go

**Comment by sayanchowdhury** [comment]
Merged #207

**Pull request closed without merging**